readme.md

```diff
@@ -3,15 +3,15 @@
 # Skipper
 
 Skipper is an HTTP router that acts as a reverse proxy (with support for flexible route definitions), alters
-requests, and responses with filters. It:
+requests, and responds with filters. It:
```

Comment:
here is the intro paragraph from the documentation (https://godoc.org/github.com/zalando/skipper), slightly modified:

'''
Skipper is an HTTP router that acts as a reverse proxy, mapping incoming requests to multiple HTTP backend services, based on routes selected by the request attributes. In between, both the requests and responses can be augmented by a filter chain defined individually for each route.
'''

Maybe we could use this in the readme, too. Does it look ok?